examples/python/VideoAPI.ipynb

```diff
@@ -0,0 +1,672 @@
+    "approx_nf = (5-2) * video.get_metadata()['video']['fps'][0]\n",
+    "print(\"We can expect approx: \", approx_nf)\n",
+    "print(\"Tensor size: \", frames[0].size())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Building a sample `read_video` function\n",
+    "\n",
+    "We can utilize the methods above to build the read video function that follows the same API to the existing `read_video` function "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def example_read_video(video_object, start=0, end=None, read_video=True, read_audio=True):\n",
+    "\n",
+    "    if end is None:\n",
+    "        end = float(\"inf\")\n",
+    "    if end < start:\n",
+    "        raise ValueError(\n",
+    "            \"end time should be larger than start time, got \"\n",
+    "            \"start time={} and end time={}\".format(s, e)\n",
+    "        )\n",
+    "    \n",
+    "    video_frames = torch.empty(0)\n",
+    "    video_pts = []\n",
+    "    if read_video:\n",
+    "        video_object.set_current_stream(\"video\")\n",
+    "        video_object.seek(start)\n",
+    "        frames = []\n",
+    "        t, pts = video_object.next()\n",
+    "        while t.numel() > 0 and (pts >= start and pts <= end):\n",
+    "            frames.append(t)\n",
+    "            video_pts.append(pts)\n",
+    "            t, pts = video_object.next()\n",
+    "        if len(frames) > 0:\n",
+    "            video_frames = torch.stack(frames, 0)\n",
+    "\n",
+    "    audio_frames = torch.empty(0)\n",
+    "    audio_pts = []\n",
+    "    if read_audio:\n",
+    "        video_object.set_current_stream(\"audio\")\n",
+    "        video_object.seek(start)\n",
+    "        frames = []\n",
+    "        t, pts = video_object.next()\n",
+    "        while t.numel() > 0 and (pts >= start and pts <= end):\n",
+    "            frames.append(t)\n",
+    "            audio_pts.append(pts)\n",
+    "            t, pts = video_object.next()\n",
+    "        if len(frames) > 0:\n",
+    "            audio_frames = torch.cat(frames, 1)\n",
+    "\n",
+    "    return video_frames, audio_frames, (video_pts, audio_pts), video_object.get_metadata()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "torch.Size([327, 3, 256, 340])"
+      ]
+     },
+     "execution_count": 43,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "vf, af, info, meta = example_read_video(video)\n",
+    "# total number of frames should be 327\n",
+    "vf.size()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "torch.Size([1024, 511])"
```

Comment:
This size for audio looks wrong. I think you might have concatenated on the wrong dimension